src/main/java/com/alkemy/ong/payloads/UserForm.java

```diff
@@ -0,0 +1,20 @@
+package com.alkemy.ong.payloads;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * @author nagredo
+ * @project OT208-server
+ * @class UserForm
+ */
+@Data
+@NoArgsConstructor
+public class UserForm {
```

Comment:
Change dto name to the pattern currently in use. Move dto to dto folder.

Reply:
Corregido en el PR OT208-35-1